README.md

```diff
@@ -1,44 +1,97 @@
-# Stripe Elements for React
-
-React components that help you quickly add Stripe Elements to your React app.
+# react-stripe-elements
 
 [![build status](https://img.shields.io/travis/stripe/react-stripe-elements/master.svg?style=flat-square)](https://travis-ci.org/stripe/react-stripe-elements)
 [![npm version](https://img.shields.io/npm/v/react-stripe-elements.svg?style=flat-square)](https://www.npmjs.com/package/react-stripe-elements)
 
+> React components for Stripe.js and Stripe Elements
+
 This project is a thin React wrapper around [Stripe.js](https://stripe.com/docs/stripe.js)
 and [Stripe Elements](https://stripe.com/docs/elements). It allows you to add Elements
 to any React app, and manages the state and lifecycle of Elements for you.
 
 The [Stripe.js / Stripe Elements API reference](https://stripe.com/docs/elements/reference)
 goes into more detail on the various customization options for Elements (e.g. styles, fonts).
 
-## Getting started
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
Could you add instructions for updating the README? (`yarn doctoc`, for example)

Reply:
👍